tests/unittests/onnxImporterTest.cpp

```diff
@@ -634,3 +634,85 @@ TEST(onnx, gather) {
   ASSERT_TRUE(gather);
   EXPECT_TRUE(gather->getResult().dims().equals({2, 4, 2}));
 }
+
+static void importSliceTest(std::string fileName, const char *inputName,
+                            const llvm::ArrayRef<size_t> inputShape,
+                            const llvm::ArrayRef<size_t> starts,
+                            const llvm::ArrayRef<size_t> outputShape) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetFilename("tests/models/onnxModels/" + fileName);
```

Comment:
Does this way of concatenating strings actually work?  It looks like it's adding two char*'s together and then making a std::string from that, which is probably not the intent...

Reply:
Well, it works in my environment, but I agree it's a bit unclean. I fixed this.

Reply:
FWIW, this is legal in standard C++ -- there's an overload of `operator+` which takes `char*` (here resulting from `"tests/models/onnxModels/"`) and `std::string` (here: `fileName`) and returns `std::string`; cf. the docs in http://eel.is/c++draft/string.op.plus#itemdecl:5 / overload (4) in https://en.cppreference.com/w/cpp/string/basic_string/operator%2B. (For example: https://wandbox.org/permlink/birDVHjYKqblSC5t). Admittedly, things would be different for `llvm::StringRef` (although there's a lightweight approach to concatenation using `llvm::Twine`, e.g., https://github.com/llvm-mirror/llvm/blob/release_70/lib/Passes/PassPlugin.cpp#L21, http://llvm.org/doxygen/classllvm_1_1Twine.html#aba6d9549c0ddfa25919915044a81eaf3).